include/libopencm3/at32/f40x/rcc.h

```diff
@@ -59,7 +59,7 @@
 #define RCC_CFGR3		MMIO32(RCC_BASE + 0x30)		/* CRM_MISC1 */
 #define RCC_CFGR4		MMIO32(RCC_BASE + 0x50)		/* CRM_MISC2 */
 #define RCC_CFGR5		MMIO32(RCC_BASE + 0x54)		/* CRM_MISC3 */
-#define RCC_INTMAP		MMIO32(RCC_BASE + 0x5c)		/* CRM_INTMAP */
+#define RCC_CFGR6		MMIO32(RCC_BASE + 0x5c)		/* CRM_INTMAP */
```

Comment:
Is this defines to be compatible with stm32, but with comments (that no-one will ever see) showing what the actual vendor names are?  I'm inclined to say you want them both defined in that case honestly.

Reply:
well, that's about two dosens registers, which do not exist in stm32f1 --  making them just as they are in at32 docs
feels sometimes really meh.  This particular diff just fixes difference in register naming with register value  define for this register below. 
And by the way comments like these _are_  wisible in doxygen, no ?